ANS_JS_template.tex

```diff
@@ -79,11 +79,11 @@
+Growing concerns over the impact of fossil fuels on climate change have driven efforts to find low emissions sources of energy. In response, fluctuating renewable energy sources, such as solar and wind power, are growing to meet more of the electricity demand. However, maintaining reliable energy accessibility to the grid requires a stable, non-fluctuating source of power. Nuclear power plants provide nearly emissions-free, reliable energy to the grid \cite{IPCC}. To best reduce reliance on fossil fuels while ensuring reliable energy generation and profitability, nuclear renewable hybrid energy systems (NRHESs) focus on tightly coupling renewable generation with a nuclear power plant by co-locating the generation sources on an industrial park. The industrial park is comprised of at least the nuclear power plant, the renewable energy source, and some form of industrial process that consumes the energy not used by the grid. In this paper, we analyze the computational modeling approaches currently being pursued for NRHES. We further investigate similarities between Nuclear Fuel Cycle Simulators (NFCS) and NRHESs to determine how NRHES development can benefit from the development of NFCSs. This paper begins by reviewing past research on NRHES to determine the necessary functionality of modeling software. After determining the necessary software capabilities for an NRHES model, we discuss the characteristics of a NFCS. The characteristics found common to both systems include the desirability of a flexible modular design, open source, able to be coupled to external pieces of software, including economic modeling, optimization methods, and sensitivity analysis as well as results which are usable to technical and non-technical people alike.
 
```

Comment:
Reviewer #4 Comment 3: I changed the conclusion to my abstract to sum up my findings that NRHES can benefit from the development of NFCSs by including some similar capabilities and characteristics.
Reviewer #4 Comment 11: Fixed first sentence
Reviewer #4 Comment 12: changed emissions-free to nearly emissions free